test/dotnet/Integration/RedisListTriggerTests.cs

```diff
@@ -149,7 +149,7 @@ public async void ListTrigger_Batch_ExecutesFewerTimes(string functionName, Type
         //[Fact]
         //public async void ListTrigger_TargetBasedScaling_E2EValidation()
         //{
-        //    string functionName = nameof(RedisListTriggerTestFunctions.ListTrigger_RedisValue_LongPollingInterval);
+        //    string functionName = nameof(ListTrigger_RedisValue_LongPollingInterval);
```

Comment:
Can we either delete or reactivate this commented test?

Reply:
ah good catch, the `RedisScalerProviderTests` contain the E2E tests for the target scaler, so this test is not necessary anymore